utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterValue.kt

```diff
@@ -9,16 +9,23 @@ import org.oppia.android.app.model.PlatformParameter
  */
 interface PlatformParameterValue<T> {
   val value: T
+  val isSynced: Boolean
 
   companion object {
     /**
      *  Creates a Platform Parameter Implementation containing the default value for a particular
      *  Platform Parameter
      */
-    fun <T> createDefaultParameter(defaultValue: T): PlatformParameterValue<T> {
+    fun <T> createDefaultParameter(
+      defaultValue: T,
+      defaultIsSynced: Boolean = false
+    ): PlatformParameterValue<T> {
       return object : PlatformParameterValue<T> {
         override val value: T
           get() = defaultValue
+
+        override val isSynced: Boolean
+          get() = defaultIsSynced
```

Comment:
```suggestion
        override val value = defaultValue
        override val isSynced = defaultIsSynced
```

Getters aren't really needed here, so this can be simplified a bit.

Reply:
I have updated this and all other instances to remove getters.